www/big-picture/welcome.spt

```diff
@@ -59,10 +59,11 @@ documentation exists for when you run into questions.
 **Fourth** is to jump in! Our main communication channel is GitHub, so if you
 don't have a GitHub account, [create
 one](https://github.com/join?source=gratipay-onboarding). Then [make a
-ticket](https://github.com/gratipay/inside.gratipay.com/issues/new) titled
-"onboard @me" (your GitHub username), and introduce yourself and what you're
-interested in working on. Your onboarding ticket will be your place to ask any
-and all questions you have as you orient yourself as a Gratipay staff member.
+ticket](https://github.com/gratipay/inside.gratipay.com/issues/new#title=onboard+@me)
```

Comment:
This part just pre-fills the title when linking out to create a new onboarding ticket.